src/locales/pl.json

```diff
@@ -279,7 +279,10 @@
     "developer_default": "The developer of this application",
     "description": {
       "only_internal": "%{name} commits to not giving the data used by the application:",
-      "internal": "%{name} commits to not giving the other data used by the application:",
+      "internal": {
+        "konnector": "Zamierzasz zainstalować łącznik między Personal Cloud a tą usługą, aby ułatwić odzyskiwanie danych. W tym celu uzyska dostęp do następujących danych osobowych, ale nigdy nikomu ich nie przekaże:",
+        "webapp": "%{name} commits to not giving the other data used by the application:"
+      },
```

Comment:
you shouldn't worry about the langage you do not understand. Community is here for that ;)